layouts/blog___news.tpl

```diff
@@ -97,7 +102,7 @@
                     {% assign article_date_format = "long" %}
                   {% endif %}
 
-                  <time class="article-date" datetime="{{ article.created_at | date: '%Y-%m-%d' }}">{{ article.created_at | format_date: article_date_format }}</time>
+                  <time class="article-date{% if article_data_show_date_defined != true %} site-data{% endif %}{% if show_article_date == false %} hide-article-date{% endif %}" datetime="{{ article.created_at | date: '%Y-%m-%d' }}">{{ article.created_at | format_date: article_date_format }}</time>
```

Comment:
Do not render `<time>` tag in public mode when `article_data_show_date_defined == false`

Should be:
```
{% if editmode or show_article_date == true %}
  <time ...
{% endif  %}
```